lib/cookie.js

```diff
@@ -620,6 +636,43 @@ Cookie.prototype.inspect = function inspect() {
     '"';
 };
 
+Cookie.prototype.toJSON = function() {
+  var obj = {};
+
+  var props = this.serializableProperties || Cookie.serializableProperties;
```

Comment:
Maybe folks want to store additional things on the `Cookie` itself, and this would allow them to add fields.

TODO: Really this is probably a set-union of the custom + standard properties, or just allow users to amend `Cookie.serializableProperties`.